### PR TITLE
Login View

### PR DESCRIPTION
**Pull request by nickcannariato**
This PR will create the login view for the Application.

## TODO
- [x] Add change handlers for the form state
- [ ] Add a check to see if the user is logged in
- [ ] Add a redirect away from the login form if the user is already logged in
- [ ] Make the form POST to the proper endpoint

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,21 +1,22 @@
 import React from "react";
-import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import { BrowserRouter as Router, Route, Switch, Redirect } from 'react-router-dom';
```
[the remark this answers is not in the document]
Comment (nickcannariato) [reply src/App.js]: We'll be handling this elsewhere 👌

File: src/views/LoginView.js [hunk 2]
```diff
@@ -1,7 +1,79 @@
 import React, { Component } from "react";
 
 export default class LoginView extends Component {
+  state = {
```
[the remark this answers is not in the document]
Comment (nickcannariato) [reply src/views/LoginView.js]: Not sure why it's not aware of the state verbiage, but oh well.

[the remark this answers is not in the document]
Comment (ptrfrncsmrph) [reply src/views/LoginView.js]: I think this is not technically part of the language yet (I think this is the proposal: https://github.com/tc39/proposal-class-fields, stage 3) but made possible by CRA's Babel process I'm assuming?

**Comment by nickcannariato** [comment]
@BrandonGardner2 @ptrfrncsmrph 

This isn't currently using the context API but it functions as a component and I think it's in a good place to merge so that we can add the context API and work with it more from there.

**Pull request merged**